dataland-frontend/src/App.vue

```diff
@@ -65,12 +65,3 @@ export default defineComponent({
   },
 });
 </script>
-
-<style lang="scss">
-@import "./assets/css/main.css";
```

Comment:
The hove colors differ:
![image](https://user-images.githubusercontent.com/47553592/210359008-2d7342d8-0ba2-45da-a40a-e930136eb30e.png)

Reply:
As discussed - let's leave as is.